packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.tsx

```diff
@@ -277,8 +280,9 @@ const FiatTransactionList = () => {
                         let account_title = wallet_title;
                         let account_currency = wallet_currency;
                         let icon = wallet_icon;
-                        let icon_type = 'fiat';
+                        let icon_type = is_crypto(wallet_currency) ? 'crypto' : 'fiat';
                         let is_deriv_apps = false;
+                        const gradient_for = 'demo';
```

Comment:
As the data in this component is mock, I have add mock-hard-coded value here, too.

Reply:
I believe you can use `is_demo` here isntead of hard coding it to `demo`

Reply:
I use hard-code `demo` value, because `is_demo` is from `tradershub-store` and it will be changed based on the selected account, but this is just mock data for demo list.

Reply:
I updated this variable.